<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SvAlarmDialog</class>
 <widget class="QDialog" name="SvAlarmDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="bnIgnore">
        <property name="text">
         <string>Игнорировать</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/alarms/icons/bullet_red.png</normaloff>:/alarms/icons/bullet_red.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageZone">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1006</width>
        <height>694</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/tree32/icons/layers_1.ico</normaloff>:/tree32/icons/layers_1.ico</iconset>
      </attribute>
      <attribute name="label">
       <string>Зона безопасности</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="vlayZone">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QWidget" name="widget" native="true"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageDetails">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1006</width>
        <height>694</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/tree32/icons/Eye.bmp</normaloff>:/tree32/icons/Eye.bmp</iconset>
      </attribute>
      <attribute name="label">
       <string>Детальная информация</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="vlayDetails"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
